cl/corpus_importer/management/commands/claims_activity_project.py

```diff
@@ -0,0 +1,265 @@
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import json
+import os
+from datetime import date
+
+import pandas as pd
+from django.conf import settings
+from django.contrib.auth.models import User
+from juriscraper.pacer import ClaimsActivity, PacerSession
+
+from cl.lib.argparse_types import valid_date
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.lib.pacer_session import get_or_cache_pacer_cookies
+from cl.search.models import Court
+
+
+def query_and_parse_claims_activity(
+    courts: list[str], date_start: date, date_end: date
+) -> None:
+    """Queries and parses claims activity for a list of courts and a specified
+     date range.
+
+    :param courts: List of courts to query the reports.
+    :param date_start: Start date for the date range to query.
+    :param date_end: End date for the date range to query.
+    :return: None, output files are stored in disk.
+    """
+
+    if not courts:
+        bankr_courts = (
+            Court.federal_courts.bankruptcy_pacer_courts().all().only("pk")
+        )
+        courts = [court.pk for court in bankr_courts]
+
+    creditor_names = {
+        "international_flavors": "International Flavors",
+        "symrise": "Symrise",
+        "givaudan": "Givaudan",
+        "firmenich": "Firmenich",
+    }
+
+    recap_user = User.objects.get(username="recap")
+    cookies = get_or_cache_pacer_cookies(
+        recap_user.pk, settings.PACER_USERNAME, settings.PACER_PASSWORD
+    )
+    s = PacerSession(cookies=cookies)
```

Comment:
Check out some of the other commands where we do client work. They take environment variables for the username/password, so we can use the client's info.

Reply:
Sure, done!